yarn.lock

```diff
@@ -347,6 +452,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/helper-simple-access@npm:^7.20.2":
```

Comment:
couldn't we consolidate with one above? it's seems to different only in minor version

Reply:
deduplicated by `yarn dedupe` I thought that we had a dedupe checker in CI
14130e270e0503b7928ec87b329977535de60eda